pom.xml

```diff
@@ -47,8 +47,8 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
     <!-- Maven Plugin for JS and CSS bundling and Unit Test execution -->
     <com.github.eirslett.frontend.version>1.6</com.github.eirslett.frontend.version>
-    <node.version>v10.4.1</node.version>
-    <npm.version>6.1.0</npm.version>
+    <node.version>v12.1.0</node.version>
+    <npm.version>6.9.0</npm.version>
 	</properties>
```

Comment:
It is ok to upgrade those for 6.2.x, but you need to keep the old ones for 6.1.x.